src/Nethermind/Nethermind.Specs.Test/ChainSpecStyle/ChainSpecBasedSpecProviderTests.cs

```diff
@@ -232,6 +232,47 @@ public void Chiado_loads_properly()
         provider.GetSpec((1, ChiadoSpecProvider.ShanghaiTimestamp)).MaxInitCodeSize.Should().Be(2 * 24576L);
     }
 
+    [Test]
+    public void Gnosis_loads_properly()
+    {
+        ChainSpecLoader loader = new(new EthereumJsonSerializer());
+        string path = Path.Combine(TestContext.CurrentContext.WorkDirectory, "../../../../Chains/gnosis.json");
+        ChainSpec chainSpec = loader.Load(File.ReadAllText(path));
```

Comment:
This is used all over those test would be good to make it a method.

Reply:
agreed, will do in next PR